src/main/java/mil/dds/anet/database/ApprovalStepDao.java

```diff
@@ -127,12 +127,7 @@ public int update(ApprovalStep as) {
 	 */
 	public boolean deleteStep(String uuid) {
 		//ensure there is nothing currently on this step
-		List<Map<String, Object>> rs = dbHandle.select("/* deleteApproval.check */ SELECT count(*) AS ct FROM reports WHERE \"approvalStepUuid\" = ?", uuid)
-				.map(new MapMapper(false))
-				.list();
-		Map<String,Object> result = rs.get(0);
-		int count = ((Number) result.get("ct")).intValue();
-		if (count != 0) {
+		if (this.isStepInUse(uuid)) {
```

Comment:
`this.` is not strictly needed.

Reply:
I am used to using 'this'. I will change it and keep it in mind.